CONTRIBUTING.md

```diff
@@ -1,6 +1,24 @@
 # C#/WinRT Contributor's Guide
 
-Below is our guidance for how to report issues, propose new features, and submit contributions via Pull Requests (PRs). Please refer to [building and running C#/WinRT](./README.md#Building-and-running-cwinrt) to get started on building the repo.
+Below is our guidance for how to build the repo, report issues, propose new features, and submit contributions via Pull Requests (PRs). 
+
+## Building the C#/WinRT repo
+
+C#/WinRT currently requires the following packages, or newer, to build:
+
+- [Visual Studio 16.8](https://visualstudio.microsoft.com/downloads/) 
+- [.NET 5.0 SDK](https://dotnet.microsoft.com/download/dotnet/5.0) 
+- [.NET Core 2.1 SDK](https://dotnet.microsoft.com/download/dotnet-core/2.1)
+- [nuget.exe 5.8.0-preview.3](https://www.nuget.org/downloads)
+- Microsoft.WinUI 3.0.0-preview1.200515.3
+
+The `build.cmd` script takes care of all related configuration steps and is the simplest way to get started building C#/WinRT. It installs prerequisites such as nuget and the .NET 5 SDK, configures the environment to use .NET 5 (creating a `global.json` if necessary), builds the compiler, and builds and executes the unit tests. To build C#/WinRT, follow these steps: 
+
+- Open a Visual Studio Developer command prompt pointing at the repo.
+- Run `build.cmd`. 
+- To launch the project in Visual Studio, run `devenv cswinrt.sln` from the same command prompt. This will inherit the necessary environment.
+
+**Note:**  By default, projection projects only generate source files for Release configurations, where cswinrt.exe can execute in seconds.  To generate projection sources for Debug configurations, set the project property GenerateTestProjection to 'true'.  In either case, existing projection sources under the "Generated Files" folder will still be compiled into the projection assembly.  This configuration permits a faster inner loop in Visual Studio.

```

Comment:
Considering rephrasing to be more clear... what do you think:

By default, the projects for various Projections [link] only generate source files for Release configurations, where `cswinrt.exe` can execute in seconds. To generate sources for the Projections on Debug configurations, set ...
This configuration permits... In either case, ...